content/kit/_index.md

```diff
@@ -26,20 +26,20 @@ With your kit, you should have the following boards:
 
 In addition to the boards, your kit will also contain:
 
-| Part                         | Quantity | Specification                 | Notes                                            |
-|------------------------------|----------|-------------------------------|--------------------------------------------------|
-| Battery Bag                  | 1        |                               |                                                  |
-| Battery Charger              | 1        | iMAX B6                       |                                                  |
-| Battery Charger Power Supply | 1        | 12V 5A                        |                                                  |
-| Battery                      | 2        | 11.1V 2200mAh Lithium Polymer |                                                  |
-| Webcam                       | 1        | TeckNet C016                  |                                                  |
-| USB Hub                      | 1        |                               |                                                  |
-| Micro USB Cable              | 2        |                               |                                                  |
-| USB B Cable                  | 1        |                               |                                                  |
-| Large CamCon                 | 2        | 7.5mm                         |                                                  |
-| Medium CamCon                | 2        | 5mm                           |                                                  |
-| Small CamCon                 | 3        | 3.81mm                        | 1 additionally found on Raspberry Pi power cable |
-| Raspberry Pi Power Cable     | 1        | Micro USB to small CamCon     |                                                  |
+| Part                                                                                                     | Quantity | Specification                 | Notes                                            |
+|----------------------------------------------------------------------------------------------------------|----------|-------------------------------|--------------------------------------------------|
+| [Battery Bag](https://hobbyking.com/en_us/lithium-polymer-charge-pack-25x33cm-jumbo-sack.html)           | 1        |                               |                                                  |
```

Comment:
Style suggestion: footnote style links _might_ make this table easier to read:

``` markdown
| [Battery Bag][battery-bag]           | 1        | ... |

[battery-bag]: https://hobbyking.com/en_us/lithium-polymer-charge-pack-25x33cm-jumbo-sack.html

Reply:
I am unable to find a reference to this syntax in the Hugo documentation. Please could you provide a link? It certainly seems like a good idea, as having very long urls in the table feels inelegant

Reply:
This style of links (search for "Reference-style links") is part of the original [markdown syntax](https://daringfireball.net/projects/markdown/syntax#link), so I don't know whether it would be documented explicitly by hugo.

Reply:
Changed in 79586e4